.bcr/config.yml

```diff
@@ -0,0 +1,6 @@
+# See https://github.com/bazel-contrib/publish-to-bcr#a-note-on-release-automation
+# Our releases are created by github actions based on pushing a tag, so they don't have an author.
+# This person's fork of BCR will be used to originate pull requests.
+fixedReleaser:
```

Comment:
Would I need to fork too since I'm the only maintainer in `.bcr/metadata.template.json`?

Reply:
no, this file is independent of the maintainers in that file. We just need to put one of us here, I'm happy to take this on but if you'd rather appear here that's fine too

Reply:
Let's keep you since you're fine with that.